test/queries/polish_00.txt

```diff
@@ -0,0 +1,199 @@
+Google Code In
+4chan
+WalkingDead
+Metal Gear Solid
+ClashOfClans
+AppStore2015
+Bernie Sanders
+Godzina Kodowania
+PairProgramming
+Thunderbird
+Hideo Kojima
+Counter Strike
+PogChamp
+Mailbox
+Dropbox
+opensource
+Elixir
+Concurrency
+Kim Jong-un
+John Oliver
+LastWeekTonight
+RickAndMorty
+Senfeild
+Hackers
+Anonimowosc
+ISIS
+Ukraina
+MetroBoomin
+Brak snu
+56Nights
+Road Rage
+Normalny dzien w Rosji
+BigBangTheory
+Hillary Clintol
+prezydencki wyscig
+Reddit
+Dota 2
+League of Legends
+BendGate
+Dirty Sprite 2
+PurpleReignTour
+Battle Royale
+peryskop
+amphidromia
+Jimmy Fallon
+Google Hangouts
+TheForceAwakens
+Spectre
+James Bond
+Ed Sheeran
+Brak pretekstu dla przemocy
+YouTubeRewind
+Bitcoin
+HipHopStarWars
+Purpose
+Justin Bieber
+California Drought
+Roman Empire
+Happy Birthday
+Crimea
+Apple Watch
+Surface Pro
+Microsoft
+Cortana
+MH370
+ipad pro
+HitTheQuan
+Young Thug
+Mozilla
+The International 5
+wykop
+RefugeeCrisis
+Kendrick Lamar
+Angela Merkel
+Donald Trump
+ataki na paryz
+13.12
```

Comment:
@PiotrKowalski What is `13.12` ? Is there a reason behind this ? I just made a sample twitter search and ended up with really wierd results. Can you explain what this is ?